package.json

```diff
@@ -63,7 +63,8 @@
   },
   "homepage": "https://github.com/software-mansion/react-native-gesture-handler#readme",
   "dependencies": {
-    "hammerjs": "^2.0.8",
+    "@egjs/hammerjs": "^2.0.17",
+    "hammerjs": "https://github.com/naver/hammer.js.git",
```

Comment:
does egjs require hammerjs to be installed as well? the docs seem to imply that only `@egjs/hammerjs` is required https://github.com/naver/hammer.js/#yarn

Reply:
I do not believe hammerjs is still needed, for this actually. Just the replacement